src/workloads/scale/LargeIndexedInsMatchingDocuments.yml

```diff
@@ -0,0 +1,84 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This is an indexed $in workload that matches documents in a collection.
+  This workload is intended to test a more representative workload
+
+  An extremelly common usecase for users is to fetch objects and hydrate with follow up $in query
+
+  For example:
+  The first query fetches an Author object
+  {
+    type: "author",
+    name: "Tyler",
+    posts: [ObjectId(1), ObjectId(2), ObjectId(3)],
+  }
+
+  The second query fetches the related, but not embedded/denormalized objects
+
+  { $in: [ObjectId(1), ObjectId(2), ObjectId(3)] }
+
+  This workload focuses on the second query of this use case matching documents
+  using arrays of integers that are generated to be used for the $in queries.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 20000
+  Update:
```

Comment:
Do you really need the **Clients** at all here?

You're not using the Update pool and you are no where near 20_000 clients elsewhere.

Note: I assume this is just cut'n'pasta from LargeIndexedIns.yml but I don't think it is required.

Reply:
By default we will be grabbing up to `maxPoolSize` clients on boot, so 20K might need to be justified

Reply:
@jimoleary is right. This is a copy/paste from `LargeIndexedIns` workload. I will remove it.